Bottom: 1f25849c7eb11958d073421cd1fce3618ff2afef
Top:    879bf48ae8eeabb7f605de89f0dc901c4953f0d6
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-03-06 22:51:31 +1300

Refresh of copy-stream

---

diff --git a/testtools/testresult/real.py b/testtools/testresult/real.py
index 5da5dd7..4268462 100644
--- a/testtools/testresult/real.py
+++ b/testtools/testresult/real.py
@@ -371,6 +371,8 @@ class CopyStreamResult(StreamResult):
     """Copies all event it receives to multiple results.
     
     This provides an easy facility for combining multiple StreamResults.
+
+    For TestResult the equivalent class was ``MultiTestResult``.
     """
 
     def __init__(self, targets):
